File: app/models/our_company.rb
class OurCompany < ActiveRecord::Base
    attr_accessible :addr, :bank_info, :en_addr, :en_name, :fax, :name, :phone, :site, :use_for_contract, :vat_info

    has_many :quotes
    has_many :contracts

    def self.available
        where("use_for_contract = 1")
    end

    def for_list_json
        attributes
    end
end

File: app/assets/javascripts/app/model/dict/OurCompany.js
Ext.define('EIM.model.dict.OurCompany', {
    extend: 'Ext.data.Model',
    fields: [
        {
            name: 'id',
            type: 'int'
        },
        {
            name: 'name',
            type: 'string'
        },
        {
            name: 'en_name',
            type: 'string'
        },
        {
            name: 'bank_info',
            type: 'string'
        },
        {
            name: 'vat_info',
            type: 'string'
        }
    ]
});
